src/io/flutter/run/daemon/DaemonJsonInputFilterProvider.java

```diff
@@ -31,9 +31,14 @@
     public List<Pair<String, ConsoleViewContentType>> applyFilter(String text, ConsoleViewContentType contentType) {
       if (!VERBOSE) {
```

Comment:
Is the VERBOSE final ever set true? Should it be removed and this code simplified?